hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -160,8 +160,19 @@ public Response getKeysForContainer(
         // Directly calling get() on the Key table instead of iterating since
         // only full keys are supported now. When we change to using a prefix
         // of the key, this needs to change to prefix seek.
-        OmKeyInfo omKeyInfo = omMetadataManager.getKeyTable(getBucketLayout())
-            .getSkipCache(containerKeyPrefix.getKeyPrefix());
+        // The first expression will try to get the OmKeyInfo object by calling
+        // getSkipCache on the BucketLayout.LEGACY table, passing in the
+        // keyPrefix as the key. If this expression returns null, meaning
+        // the key was not found in the LEGACY table, the second expression
+        // will be evaluated and the OmKeyInfo will be obtained from the
+        // BucketLayout.FILE_SYSTEM_OPTIMIZED table.
```

Comment:
Can we have a simple comment like, 
`Directly calling getSkipCache() on the table based on the bucket layout(file table for FSO buckets and key table for others) instead of iterating since only full keys are supported now.`

Reply:
Thanks for the review. I have trimmed down the comment !!